README.md

```diff
@@ -13,44 +15,44 @@ After installing the template, you can quickly create the following items:
 * masafx-service: MASA Framework Pure Template (Minimal Api)
 * masafx-service-cqrs: MASA Framework CQRS Template(Minimal Api、CQRS)
 
-## Install Or Upgrade Template
-
-open windows terminal and run commands:
+## Install template
 
 ```shell
 dotnet new install Masa.Template
 ```
 
-## Get Version
-
-open windows terminal and run commands:
+### Install specific version or prerelease version
 
 ```shell
-dotnet new uninstall
```

Comment:
It is necessary to check the current template version

Reply:
I forgot the template name!

Reply:
version is not necessary